data/common.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-letsencrypt::configure_epel: false
+{}
```

Comment:
my personal preference is to delete the file. when I see a yam file I assume it has content

Reply:
Agreed, deleted `common.yaml`.